QSB/GeyserSync/QSBGeyser.cs

```diff
@@ -0,0 +1,42 @@
+﻿using QSB.Events;
+using QSB.WorldSync;
+using UnityEngine.Networking;
+
+namespace QSB.GeyserSync
+{
+    public class QSBGeyser : WorldObject
+    {
+        public override string UniqueName => _geyserController.name;
```

Comment:
had `_geyserController.name` not been unique, we'd use `_geyserController.gameObject.GetHierarchy();`